pkg/apis/core/types.go

```diff
@@ -5628,7 +5655,8 @@ type TopologySpreadConstraint struct {
 	// We consider each <key, value> as a "bucket", and try to put balanced number
 	// of pods into each bucket.
 	// We define a domain as a particular instance of a topology.
-	// Also, we define an eligible domain as a domain whose nodes match the node selector.
+	// Also, we define an eligible domain as a domain whose nodes meet the node affinity and
+	// node taints' constraints by respecting NodeInclusionPolicy.
```

Comment:
```suggestion
	// Also, we define an eligible domain as a domain whose nodes meet the nodeInclusionPolicy.
```

Although now it will be 2 fields.